parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -519,8 +512,8 @@ static Context deleteContext(Map<String, String> config) {
             compressionLevel,
             rowGroupCheckMinRecordCount,
             rowGroupCheckMaxRecordCount,
-            bloomFilterMaxBytes,
-            columnBloomFilterEnabled,
+            PARQUET_BLOOM_FILTER_MAX_BYTES_DEFAULT,
+            ImmutableMap.of(),
```

Comment:
for most use cases, this change makes sense.

If some datasets have high updates rate and generates a lot of large delete files. would the bloom filter for delete file be useful too?

if yes, we can introduce a config to enable/disable bloom filter for delete files only. If not, this change is good to me.

@huaxingao @hililiwei @chenjunjiedada

Reply:
or delete files are expected to be compacted/consolidated anyway. Hence the bloom filter on delete files never makes sense.

Reply:
> If some datasets have high updates rate and generates a lot of large delete files. would the bloom filter for delete file be useful too?
 That should produce position deletes

Reply:
> If some datasets have high updates rate and generates a lot of large delete files. would the bloom filter for delete file be useful too?

First, we don't have filter logic for the bloom filter right now.  Second, high-rate updates mostly produce position delete which doesn't contains a column to build a bloom filter unless the upstream is deleting a set of records. In that case, the records of the filtered data files should always pass the bloom filter of equality delete.

> delete files are expected to be compacted/consolidated anyway. Hence the bloom filter on delete files never makes sense.

I think so, the deletes impact read performance, as far as I know, all real productions need the async tasks to compact them to achieve good performance.